src/DotVVM.Framework/ResourceManagement/ResourceRepositoryJsonConverter.cs

```diff
@@ -36,7 +37,7 @@ protected virtual IEnumerable<Type> ResolveAllTypesDerivedFromIResource(string d
 
         protected virtual IEnumerable<Assembly> GetAllAssembliesLoadedAssemblies()
         {
-            return ReflectionUtils.GetAllAssemblies();
+            return AppDomain.CurrentDomain.GetAssemblies();
```

Comment:
It seems to be unused, anyway. I don't know why it's virtual, though.

Reply:
Removed together with `ResolveAllTypesDerivedFromIResource` that was unused too.

Reply:
Compiler overrides this method for its purposes.